db/migrate/20180921170100_add_default_reference_to_stages.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultReferenceToStages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :stages, :default_reference, :string
```

Comment:
`default: 'master'`

Reply:
Do we always want a default though? Might be kind of annoying for people to have to delete "master" each time.